activerecord/lib/active_record/database_configurations.rb

```diff
@@ -90,6 +90,10 @@ def empty?
     end
     alias :blank? :empty?
 
+    def any?
```

Comment:
Can you add the documentation?

Reply:
If we use the `delegate to:` approach, is it not possible to add documentation? Is that the reason to spell it out like this?

Reply:
Using the delegate approach is better indeed, I switched to that.
Regarding the documentation, I don't think we need that, since this is just to make it backward compatible, this behaviour existed before with the hash config.